src/utilities/theme/tests/utils.test.ts

```diff
@@ -1,53 +1,120 @@
+      '--icon-on-branded': 'hsl(0, 0%, 98%, 1)',
+      '--icon-on-dark': 'hsl(210, 9%, 98%, 1)',
+      '--icon-on-light': 'hsl(210, 9%, 18%, 1)',
+      '--icon-on-opposite': 'hsl(210, 9%, 98%, 1)',
+      '--icon-on-surface': 'hsl(210, 9%, 18%, 1)',
+      '--interactive': 'hsl(210, 9%, 13%, 1)',
+      '--interactive-action': 'hsl(210, 93%, 44%, 1)',
+      '--interactive-action-disabled': 'hsl(210, 93%, 58%, 1)',
+      '--interactive-action-hovered': 'hsl(210, 93%, 37%, 1)',
+      '--interactive-action-muted': 'hsl(210, 93%, 51%, 1)',
+      '--interactive-action-pressed': 'hsl(210, 93%, 31%, 1)',
+      '--interactive-focus': 'hsl(210, 93%, 58%, 1)',
+      '--interactive-neutral-elevation-0': 'hsl(210, 9%, 100%, 1)',
+      '--interactive-neutral-elevation-1': 'hsl(210, 9%, 94%, 1)',
+      '--interactive-neutral-elevation-2': 'hsl(210, 9%, 92%, 1)',
+      '--interactive-neutral-elevation-3': 'hsl(210, 9%, 86%, 1)',
+      '--interactive-neutral-elevation-4': 'hsl(210, 9%, 76%, 1)',
+      '--interactive-neutral-elevation-5': 'hsl(210, 9%, 66%, 1)',
+      '--interactive-selected': 'hsl(210, 93%, 96%, 1)',
+      '--interactive-selected-hovered': 'hsl(210, 93%, 89%, 1)',
+      '--interactive-selected-pressed': 'hsl(210, 93%, 82%, 1)',
+      '--on-surface': 'hsl(210, 9%, 13%, 1)',
+      '--success': 'hsl(165, 100%, 25%, 1)',
+      '--success-divider': 'hsl(165, 100%, 25%, 1)',
+      '--success-icon': 'hsl(165, 100%, 25%, 1)',
+      '--success-surface': 'hsl(165, 100%, 88%, 1)',
+      '--success-surface-muted': 'hsl(165, 100%, 98%, 1)',
+      '--success-text': 'hsl(165, 100%, 15%, 1)',
+      '--surface': 'hsl(0, 0%, 98%, 1)',
+      '--surface-background': 'hsl(0, 0%, 98%, 1)',
+      '--surface-foreground': 'hsl(0, 0%, 100%, 1)',
+      '--surface-foreground-subdued': 'hsl(0, 0%, 90%, 1)',
+      '--surface-opposite': 'hsl(0, 0%, 0%, 1)',
+      '--text-disabled-on-dark': 'hsl(210, 9%, 80%, 1)',
+      '--text-disabled-on-light': 'hsl(210, 9%, 63%, 1)',
+      '--text-disabled-on-opposite': 'hsl(210, 9%, 80%, 1)',
+      '--text-disabled-on-surface': 'hsl(210, 9%, 63%, 1)',
+      '--text-muted-on-branded': 'hsl(0, 0%, 90%, 1)',
+      '--text-muted-on-dark': 'hsl(210, 9%, 90%, 1)',
+      '--text-muted-on-light': 'hsl(210, 9%, 38%, 1)',
+      '--text-muted-on-opposite': 'hsl(210, 9%, 90%, 1)',
+      '--text-muted-on-surface': 'hsl(210, 9%, 38%, 1)',
+      '--text-on-branded': 'hsl(0, 0%, 100%, 1)',
+      '--text-on-dark': 'hsl(210, 9%, 100%, 1)',
+      '--text-on-light': 'hsl(210, 9%, 13%, 1)',
+      '--text-on-opposite': 'hsl(210, 9%, 100%, 1)',
+      '--text-on-surface': 'hsl(210, 9%, 13%, 1)',
+      '--warning': 'hsl(39, 100%, 66%, 1)',
+      '--warning-divider': 'hsl(39, 100%, 66%, 1)',
+      '--warning-icon': 'hsl(39, 100%, 66%, 1)',
+      '--warning-surface': 'hsl(39, 100%, 88%, 1)',
+      '--warning-surface-muted': 'hsl(39, 100%, 98%, 1)',
+      '--warning-text': 'hsl(39, 100%, 30%, 1)',
```

Comment:
These values will more than likely change, but testing the output doesn't hurt now.